examples/AspNet6/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,55 @@
+// <copyright file="WeatherForecastController.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace Examples.AspNet6.Controllers;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+
+[ApiController]
+[Route("[controller]")]
+public class WeatherForecastController : ControllerBase
+{
+    private static readonly string[] Summaries = new[]
+    {
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching",
+    };
+
+    private readonly ILogger<WeatherForecastController> logger;
+    private readonly AspNet6Meter aspNet6Meter;
+
+    public WeatherForecastController(ILogger<WeatherForecastController> logger, AspNet6Meter aspNet6Meter)
+    {
+        this.logger = logger;
+        this.aspNet6Meter = aspNet6Meter;
+    }
+
+    [HttpGet(Name = "GetWeatherForecast")]
+    public IEnumerable<WeatherForecast> Get()
+    {
+        using var activity = Tracing.ActivitySource.StartActivity("WeatherForecast GET", ActivityKind.Internal);
+        activity?.SetStartTime(DateTime.UtcNow);
+        this.aspNet6Meter.Requests.Add(1, KeyValuePair.Create<string, object?>("Method", "WeatherForecast GET"));
```

Comment:
lets use a more useful example - Request Count is something already provided by the ASP.NET Instrumentation.